planet/cli/data.py

```diff
@@ -14,17 +14,20 @@
 """The Planet Data CLI."""
 from typing import List, Optional
 from contextlib import asynccontextmanager
+from pathlib import Path
 
 import click
 
+from planet.reporting import StateBar
 from planet import data_filter, DataClient, exceptions
-from planet.clients.data import (LIST_SEARCH_TYPE,
+from planet.clients.data import (SEARCH_SORT,
```

Comment:
These are mixed up due to some weird merge conflict resolution, sorry about that :)